sources/Context.cpp

```diff
@@ -17,11 +17,26 @@
 
 #include "Context.h"
 
+#include "Log.h"
+
 #include <tuple>
 
 namespace dfl {
-Context::Context(const std::string& networkFilepath) : networkManager_(networkFilepath) {
-  networkManager_.onNode(std::bind(&Context::processSlackNode, this, std::placeholders::_1));
+Context::Context(const std::string& networkFilepath, const std::string& configFilepath) :
+    networkManager_(networkFilepath),
+    config_(configFilepath),
+    slack_node_{} {
+  auto found_slack_node = networkManager_.getSlackNode();
+  if (found_slack_node.is_initialized() && !config_.isAutomaticSlackBusOn()) {
+    slack_node_ = *found_slack_node;
+  } else {
+    // slack node not given in iidm or not requested: it is computed internally
+    if (!found_slack_node.is_initialized() && !config_.isAutomaticSlackBusOn()) {
+      // TODO(lecourtoisflo) use warning without restriction for language
+      LOG(warn) << "Network slack node requested but not present in network input file" << LOG_ENDL;
+    }
+    networkManager_.onNode(std::bind(&Context::processSlackNode, this, std::placeholders::_1));
```

Comment:
Would be good to display the final slack node and potentially its origin (automatic, from iidm ...)

Reply:
(we can start with an english message and add later dictionaries)

Reply:
Will be done in future patch (when dictionnary is added)